composer.json

```
@@ -9,15 +9,17 @@
       "*": "dist"
     }
   },
+  "prefer-stable": true,
+  "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^11.0",
+    "pimcore/pimcore": "^11.x-dev",
```

```suggestion
    "pimcore/pimcore": "^11.0",
```
Any specific reason for requiring x-dev?

---

No, this was just for some testing. Thx for the catch.